hipify-clang/src/HipifyAction.cpp

```diff
@@ -371,7 +373,7 @@ std::unique_ptr<clang::ASTConsumer> HipifyAction::CreateASTConsumer(clang::Compi
         ).bind("cudaBuiltin"),
         this
     );
-
+    */
```

Comment:
Let's just remove the dead code rather than comment it - we can always retrieve from Git revision history if desired.

Reply:
Done, verified.